modules/drivers/redshift/src/metabase/driver/redshift.clj

```diff
@@ -92,6 +93,19 @@
 ;;; |                                           metabase.driver.sql impls                                            |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+;; custom Redshift type handling
+
+(def ^:private database-type->base-type
+  (sql-jdbc.sync/pattern-based-database-type->base-type
+    [
+     [#"(?i)CHARACTER VARYING" :type/Text]       ; Redshift uses CHARACTER VARYING (N) as a synonym for VARCHAR(N)
+     [#"(?i)NUMERIC"           :type/Decimal]])) ; and also has a NUMERIC(P,S) type, which is the same as DECIMAL(P,S)
+
+(defmethod sql-jdbc.sync/database-type->base-type :redshift
+           [driver column-type]
+           (or (database-type->base-type column-type)
+               ((get-method sql-jdbc.sync/database-type->base-type :postgres) driver column-type)))
```

Comment:
This should only be indented 2 spaces... see all the other places we use `defmethod` https://guide.clojure.style/#body-indentation

Reply:
Fixed.